app/(authenticated)/(content)/embedded-portal/embedded-portal.tsx

```diff
@@ -86,7 +86,10 @@ export default function EmbeddedPortal({
             <div className="mt-8">
               <div className="content">
                 <div>
-                  <Button className="contrast" onClick={onOpenSpace}>
+                  <Button
+                    className="contrast sm:w-auto md:w-2/3 lg:w-3/4 h-14"
```

Comment:
Can you apply these styles to the base classes in the Button component so every Button has the same?